.github/workflows/unittests.yml

```diff
@@ -41,21 +50,21 @@
            environment-file: ${{ matrix.environment-file }}
            micromamba-version: 'latest'
       
-       - name: install bleeding edge libpysal & esda (only Ubuntu / Python 3.10)
+       - name: install bleeding edge libpysal & esda (Ubuntu / Python 3.10)
          shell: bash -l {0}
          run: |
-           pip install git+https://github.com/pysal/libpysal.git@master
-           pip install git+https://github.com/pysal/esda.git@master
+           ${{ env.PIP_INSTALL }}libpysal.git@master
+           ${{ env.PIP_INSTALL }}esda.git@master
```

Comment:
While I understand that this saves a few characters, the original version is much easier to follow imho. With the new one, I have to think what the line does, with the original one I don't.

Reply:
Fair enough. I'll revert.